ci/builders/local_engine.json

```diff
@@ -65,6 +65,40 @@
         }
       }
     },
+    {
```

Comment:
These two configs don't need any changes, but they should probably live in `mac_unopt.json`. The idea is that the builds in this file run on CI only to keep the RBE cache warm, and so that `et` has nice names for builds commonly used in local development. By contrast, the builds in `mac_unopt.json` are for running tests on CI (or builds like this, where the build is essentially the test).

Reply:
The `_unopt` in the file name threw me off since these builds are optimized. It's fine if I add the opt variants to the file right?

Reply:
Yes, it's fine to add the opt variants. The `_unopt.json` files contain misc. tests, and the name is a relic. There's also no better place for these new builds to go right now.

Reply:
Ok, done.